.pipelines/e2e-tests-template.yaml

```diff
@@ -48,11 +48,34 @@ jobs:
           parameters:
             identitySubscriptionID: $(SUBSCRIPTION_ID)
 
+        - script: |
+            #This gets latest released chart(tgz) version from charts directory
+            export HELM_CHART_DIR=$(ls -h charts/*.tgz | sort --version-sort --field-separator=- --key=6 | tail -n 1)
+
+            #Image version of latest released chart
+            IMAGE_VERSION=0.0.14
```

Comment:
if this is always latest, can we get this automatically?

Reply:
This is targeted for e2e testing of backward compatibility and released helm chart. The other option is to rely on default values in values.yaml of released chart(which we manually updates while releasing)